<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>16</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumGothic</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status Label</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>391</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumGothic</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWarn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>391</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumGothic</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WARN</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
